source/_components/sensor.zoneminder.markdown

```diff
@@ -27,8 +27,19 @@ To set it up, add the following information to your `configuration.yaml` file:
 sensor:
   - platform: zoneminder
     include_archived: false
+    monitored_conditions:
```

Comment:
We try to keep the examples as minimal as possible, by limiting the number of configuration option in the examples.

Please remove this added (optional) configuration option.